### PR TITLE
Reset bold attribute

### PR DESCRIPTION
**Pull request by serge-v**

### CODE REVIEW

File: log/term/colorlogger.go [hunk 1]
```diff
@@ -42,15 +42,15 @@ const (
...
 	fgColorBytes    [][]byte
 	bgColorBytes    [][]byte
 )
 
 func init() {
 	// Default
-	fgColorBytes = append(fgColorBytes, []byte("\x1b[39m"))
-	bgColorBytes = append(bgColorBytes, []byte("\x1b[49m"))
+	fgColorBytes = append(fgColorBytes, []byte("\x1b[39;22m"))
+	bgColorBytes = append(bgColorBytes, []byte("\x1b[49;22m"))
```
Comment (ChrisHines) [inline log/term/colorlogger.go]: Is this change required? I believe we maintain the invariant that `resetColorBytes` is emitted at the end of each log record, which implies that no color codes are in effect at the point that these sequences are applied, making the additional `22m` code redundant with the change on line 45. Thoughts?

Comment (serge-v) [inline log/term/colorlogger.go]: you are right, it's not needed. Only in resetColorBytes.

**Comment by serge-v** [comment]
will redo
